README.md

```diff
@@ -177,71 +176,68 @@ print "screenShot successfully generated :".$chrome->getScreenShot("/tmp/hello/t
 
 ## Setting options
 
-The `google-chrome` shell command accepts different types of options:
-for complete list of options you can visit
+The `google-chrome` shell command accepts different types of options, 
+for the complete list of options you can visit
 https://peter.sh/experiments/chromium-command-line-switches/
 
 
 
 ### Wrapper Methods
 
 
- * `constructor`: Accepts $url to visit(for pdf/screenshot) as first parameter
- you can pass this as null and later use `setUrl`, and second parameter is binary
- path of google-chrome installed in your system as second parameter if no binaryPath is provided
- is uses default location `/usr/bin/google-chrome`
- but you still can provide binary path later using `setBinaryPath`,
- constructor also put some default arguments like
- `headless , disable-gpu` which are necessary for google-chrome to work on cli
- * `setBinaryPath` which accepts binary path and set it for you
- * `setArguments` to set options of google-chrome it accepts array of options in a format
+ * `constructor`: Accepts the $url to visit(for pdf/screenshot) as the first parameter.
+ You can pass this as null and later use `setUrl`. The second parameter is the binary
+ path of google-chrome installed in your system. If no binaryPath is specified
+ it uses the default location `/usr/bin/google-chrome`.
+ You can still provide a binary path later using the `setBinaryPath`
+ constructor. Default arguments like
+ `headless, disable-gpu` are necessary for google-chrome to work on cli
+ * `setBinaryPath` Accepts a binary path and sets it for you
+ * `setArguments` sets options of google-chrome, it accepts an array of options in a format
```

Comment:
1:please change it to `arguments of google-chrome`
2:it accepts an array if `arguments`

Reply:
check other parts of this file to , replace options with arguments where it's necessary

thanks

Reply:
Thanks for the feedback, I've made the change you requested. Please let me know if that's sufficient or if I should modify it in some other way.